src/locales/en/translation.json

```diff
@@ -26,7 +26,7 @@
     },
     "mnemonic": {
       "header": "Enter your keyphrase",
-      "instruction": "Enter your 12, 18 or 24 words keyphrase below, each words separated by spaces.",
+      "instruction": "Enter all your keyphrase words below, separated by spaces. Most keyphrases are made of either 24 words, or 12 words.",
```

Comment:
Also please update French translation (kudos to Raphael):
```
Entrez ci-dessous tous les mots qui composent votre phrase secrète (séparés par des espaces). La plupart des phrases secrètes sont composées de 24 ou 12 mots.
```

Reply:
Yay